docs/product-editor-development/faq.md

```diff
@@ -0,0 +1,11 @@
+# Frequently Asked Questions
+
+> ⚠️ **Notice:** This documentation is currently a **work in progress**. While it's open to the public for transparency and collaboration, please be aware that some sections might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
+
+## Why is my block not showing up?
+
+## Why is my custom block not showing up?
+
+## Why is my block not editable?
+
+## Why is my block not saving its value?
```

Comment:
I'd leave out docs that are placeholders

Reply:
I agree here, let's keep this items for a follow-up.